doc/source/whatsnew/v0.25.0.rst

```diff
@@ -169,7 +169,7 @@ Performance Improvements
 - Improved performance of :meth:`Series.searchsorted`. The speedup is especially large when the dtype is
   int8/int16/int32 and the searched key is within the integer bounds for the dtype (:issue:`22034`)
 - Improved performance of :meth:`pandas.core.groupby.GroupBy.quantile` (:issue:`20405`)
-
+- Improved performance of :meth:`read_csv` (faster tokenizing and faster parsing of small float numbers)
```

Comment:
can you add this issue number (the PR number as there is no issue). 

Improved performance of :meth:`read_csv` by faster tokenizing and faster parsing of small float numbers. (:issue:`....`)

Reply:
Fixed